stable/rabbitmq-ha/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 name: rabbitmq-ha
 apiVersion: v1
 appVersion: 3.7.12
-version: 1.25.0
+version: 1.25.1
```

Comment:
New feature -> `1.26.0`. Other than that it looks good.

Reply:
Done.